CONTRIBUTORS

```diff
@@ -154,4 +155,4 @@ R. Usamentiaga
 Dmytro Shaurin
 Rustam Sayfutdinov
 Konstantin Stukov <havendv@gmail.com>
-jorgectf
\ No newline at end of file
+jorgectf
```

Comment:
sorry about this, I don't know how this got in here

Reply:
Your editor probably put a new line at the end on saving: it's not a problem.